### PR TITLE
Consistent chip rendering in tables

### PR DESCRIPTION
**Pull request by snopoke**
### Product Description

Participant and record chips in the session, all-sessions and trace tables render consistently: one chip style, one line each, with the full label in a tooltip when it's too long to fit. A participant whose name *is* their email no longer shows it twice (`skelly@dimagi.com (skelly@dimagi.com)`). Separately, the last few user-facing "Experiment" labels now say "Chatbot".

### Technical Description

Chips had two render paths that had drifted apart: `generic/chip.html` (used by `render_*` methods) baked in `btn-soft btn-primary`, while `chip_action` renders through `generic/action.html`, which falls back to `btn-ghost` unless the caller passes `button_style` — `ChatbotTable` did, `TraceTable` didn't. The style is now a single `actions.CHIP_BUTTON_STYLE` constant, and truncation lives in one partial (`generic/chip_label.html`) that both paths include.

Worth knowing for review:

- `chip_action` serves two roles — chips that stand in for the record (trace bot/session, chatbot name, queue name) and chips that read as buttons (`Session Details`, `View Session`, `Refresh`, `Annotate`). I deliberately did **not** change `chip_action`'s default style, so the buttons keep their ghost look; only the identity chips opt in. That's why the constant is passed at the call site rather than defaulted.
- `Action.get_context` now sets `truncate: False` explicitly. Without it, an action inherits whatever `truncate` happens to be in the surrounding template context.
- Truncation caps chips at `max-w-xs`; `min-w-0` on the label span is load-bearing, since a flex item won't shrink below the width of an unbreakable string like an email address.

The "Chatbot" rename is its own commit (`565667ff6`) and is pure label text. One of those edits made a singular per-row column header plural; I made it singular again to match the identical header in `admin/sections/whatsapp.html`.

### Demo

Drove all three tables locally with participants covering each identifier shape — anonymous, bare phone number, email-as-name, name plus email, and a deliberately over-long email. Every chip renders on one line at the same height; the over-long one ellipsizes at 20rem without widening the column.

### Docs and Changelog
- [ ] This PR requires docs/changelog update

### CODE REVIEW

File: templates/generic/action.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% if action_url %}
-  <a {% if not disabled %}href="{{ action_url }}"{% endif %} class="btn btn-sm join-item {{ button_style|default_if_none:"btn-ghost" }}{% if disabled %} pointer-events-none{% endif %}" {% if disabled %}aria-disabled="true"{% endif %} {% if title %}title="{{ title }}"{% endif %} {% if not disabled and open_url_in_new_tab %}target="_blank" rel="noopener noreferrer"{% endif %}>
-    {{ label }}
+  <a {% if not disabled %}href="{{ action_url }}"{% endif %} class="btn btn-sm join-item {{ button_style|default_if_none:"btn-ghost" }}{% if truncate %} max-w-xs{% endif %}{% if disabled %} pointer-events-none{% endif %}" {% if disabled %}aria-disabled="true"{% endif %} {% if title %}title="{{ title }}"{% endif %} {% if not disabled and open_url_in_new_tab %}target="_blank" rel="noopener noreferrer"{% endif %}>
+    {% include "generic/chip_label.html" %}
```
[the remark this answers is not in the document]
Comment (snopoke) [reply templates/generic/action.html]: Correct that `default_if_none` was inert here, but the cause is upstream: `chip_action` defaults `button_style` to `""` and `Action.get_context` only sets the key when truthy, so the variable is *missing*, not `None` — the fallback has never applied. Verified against rendered output: the attribute comes out as `class="btn btn-sm join-item "` with zero occurrences of `btn-ghost`.

Switching to `default` would therefore not restore intended styling, it would newly apply ghost styling to every action button (Session Details, View Session, Refresh, edit/delete icons) — a visual change beyond this PR's scope. Instead dropped the dead fallback in a4f050554, across all four action templates that carried it (`action.html`, `action_ajax.html`, `action_modal.html`, `continue_chat_action.html`). Rendered output is byte-identical.

*— Claude Code*

File: apps/human_annotations/tables.py [hunk 2]
```diff
@@ -72,7 +73,7 @@ class AnnotationItemTable(tables.Table):
             chip_action(
                 label="Annotate",
                 url_factory=_item_chip_url,
-                button_style="btn-soft btn-primary",
+                button_style=actions.CHIP_BUTTON_STYLE,
```
[the remark this answers is not in the document]
Comment (snopoke) [reply apps/human_annotations/tables.py:76]: Not applicable — `button_style="btn-soft btn-primary"` was already on this action before the PR (see `main`: apps/human_annotations/tables.py:75). This commit only replaced the literal with the shared constant, so the Annotate chip is unchanged. Removing it would alter existing UI rather than preserve it.

*— Claude Code*

**Review by ocs-percy — APPROVED** [review]
Clean UI consistency fix.

- `CHIP_BUTTON_STYLE` constant is the right call — removes the drift risk that caused the style gap in `TraceTable` in the first place.
- `chip_label.html` as the single truncation partial means both rendering paths stay in sync going forward.
- `truncate: False` in `Action.get_context` is good defensive programming; template context leakage is exactly the kind of thing that causes subtle bugs.
- `default_if_none:"btn-ghost"` removal is safe — Django templates resolve undefined to `""`, not `None`, so the filter never fired.
- `Participant.__str__` duplicate-display fix is correct and well-tested across all identifier shapes.
- Deliberate choice not to default `truncate=True` in `chip_action` is right — identity chips and button chips are different things.
- The `min-w-0` annotation in `chip_label.html` is accurate and will save the next person from wondering why `truncate` is not working on email addresses.

🚚
